lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -422,7 +422,7 @@ def test_query_counts(self, store_type, mongo_count):
 
         RequestCache.clear_all_namespaces()
 
-        expected_query_count = 49
+        expected_query_count = 51
```

Comment:
just a small description of why this count bump is necessary, in case anyone isn't aware.

Reply:
2 additional queries for `course_grade_report_verified_only`; It seems there are two calls instead of one when we check for any active courseoverride waffleflag.